test/sales_analyst_test.rb

```diff
@@ -20,76 +20,151 @@ def setup
     @sales_analyst = sales_engine.analyst
   end
```

Comment:
We'll need to fix all the spacing between methods here.

Reply:
agreed